flexmeasures/cli/data_add.py

```diff
@@ -1747,34 +1748,43 @@ def add_toy_account(kind: str, name: str):
         **MsgStyle.SUCCESS,
     )
 
-    def create_power_asset(
-        asset_name: str, asset_type: str, sensor_name: str, **attributes
+    account_id = user.account_id
+    print(account_id)
+
+    def create_sensor_asset(
+        asset_name: str,
+        asset_type: str,
+        sensor_name: str,
+        unit: str = "MW",
+        **attributes,
     ):
         asset = get_or_create_model(
             GenericAsset,
             name=asset_name,
             generic_asset_type=asset_types[asset_type],
-            owner=user.account,
+            owner=Account.query.get(account_id),
```

Comment:
This is because the `Account` object disappears from the SQLAlchemy session when entering the `Reporter`. This has happened before with the `Scheduler` as well and I think it gets lost in the process of serializing/deserializing IDs into objects.